src/components/Header.js

```diff
@@ -1,15 +1,46 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
```

Comment:
Space between third party imports and svgs and local imports.

Generally it should look like this:

```
// 3rd party imports

// svg imports

// local imports

// css imports
```

Each group with a space in between. Fix here and throughout the app